tests/scripts/tg/run_tg_frequent_tests.sh

```diff
@@ -7,6 +7,7 @@ run_tg_tests() {
   echo "LOG_METAL: running run_tg_frequent_tests"
 
   pytest tests/ttnn/multichip_unit_tests/test_multidevice_TG.py
+  pytest models/demos/t3000/llama2_70b/tests/test_llama_mlp_galaxy.py
```

Comment:
What's the difference between t3000 and TG system?
are these interchangable?
i'm not sure if it's good to be calling t3k tests or putting galaxy tests in t3k folders if there's no difference

Reply:
TG system is a the 32-chip instead of the 8-chip for t3000. We are sort of path-clearing for other model teams to move to galaxy eventually so there should definitely be a TG folder when the move is ready, for now we can either put it under our common llama folder or with the rest of the TG tests

Reply:
if this is TG specific, there should a folder for that. and common model functions should be in its own directory so product specific scripts can import that.

A structure like this would allow users to just invoke any pytests within a particular folder which would simplify a lot of infra and tests workflows.

Open to suggestions but at the very least, I think putting it in its own TG folder would be the bare minimum here

Reply:
Yes, I agree. We need a TG folder folder. Keeping the tests under t3000 will create confusion.